src/plugins/ear-training/info.js

```diff
@@ -11,12 +12,13 @@ export default {
         startAbcCode: 'X:1',
         fullAbcCode: 'X:1',
         sound: {
-          type: 'midi',
+          type: SOUND_TYPE.midi,
           url: null,
           text: null
         }
       }
     ]
   }),
-  cloneContent: content => cloneDeep(content)
+  cloneContent: content => cloneDeep(content),
+  getCdnResources: content => content.tests.filter(test => test.sound?.type === SOUND_TYPE.internal && test.sound.url).map(test => test.sound.url)
```

Comment:
Small thing: can `content.tests` be `null`/`undefined` or is it always `[]`?

Reply:
It's always `[]`, created by the default content.